pypeit/calibrations.py

```diff
@@ -301,82 +301,48 @@ def get_align(self):
         """
         Load or generate the alignment frame
 
-        Requirements:
-           master_key, det, par
+        Requires: :attr:`slits`, :attr:`det`, :attr:`par`
 
         Returns:
-            ndarray or str: :attr:`align`
+            :class:`pypeit.alignframe.Alignments`:
 
         """
         # Check for existing data
         if not self._chk_objs(['msbpm', 'slits']):
-            msgs.error("Don't have all the objects")
+            msgs.warn('Must have the bpm and slits to make the alignments!  Skipping and may crash down the line')
```

Comment:
If this is true (i.e, you need these things to proceed) I think this should crash instead of just throwing a warning.

Reply:
I feel like it's better to throw an error at the place where the problem occurs, but I was just following suit with `get_flats`, `get_tilts`, and `get_wv_calib` in `calibrations.py`. I've made the change in this instance, but should we also change the other methods as well?

Reply:
Sorry, didn't realize we were doing this elsewhere.  @profxj what do you suggest?